README.md

```diff
@@ -4,4 +4,4 @@ Idle game by Kars and Zimmy
 
 This is a change to commit
 
-Push commit test #1
\ No newline at end of file
+Kars Branch?
```

Comment:
I think this is a kars branch? shoudl we add it

Reply:
no